packages/common/core-interfaces/src/fluidRouter.ts

```diff
@@ -25,23 +25,3 @@ export interface IResponse {
 	headers?: { [key: string]: any };
 	stack?: string;
 }
-
```

Comment:
What about the interfaces above? Are they still used?

Reply:
These interfaces are used in other flows like `resolveHandle` or `attach`